contracts/L2/ETHxPoolV5.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.22;
+
+import { ReentrancyGuardUpgradeable } from "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
+import { AccessControlUpgradeable } from "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import { SafeERC20, IERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/**
+ * @title ETHxPool Contract
+ * @author Stader Labs
+ * @notice This contract is responsible for the swap of ETHx; the user must deposit ETH in order to receive ETHx in
+ * return.
+ */
+interface AggregatorV3Interface {
+    function latestRoundData()
+        external
+        view
+        returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound);
+}
+
+contract ETHxPoolV5 is AccessControlUpgradeable, ReentrancyGuardUpgradeable {
```

Comment:
Could we rename this as ETHxPoolV1 ?

Reply:
[487ee50](https://github.com/stader-labs/ethx_oft/pull/8/commits/487ee50f73764c2e76e40af2f3d8eeeffbb9abc4)